Packs/illuminate/ReleaseNotes/1_1_13.md

```diff
@@ -0,0 +1,13 @@
+
+#### Integrations
+
+##### Analyst1
+
```

Comment:
the release note does not meet our requirements
for added items, the note needs to start with added ....
if item is updated, to start with updated ...
for improve, improved ...
you can see the exact expected template by running `demisto-sdk doc-review --templates`
or delete this rn file and run the command demisto-sdk update-release-notes -g, it will generate a general template to edit